app/src/renderer/components/common/TmBtn.vue

```diff
@@ -1,36 +1,32 @@
 <template>
-  <router-link v-if="type === 'link'" :to="to" class="tm-btn" exact="exact"
-    ><span :class="btnClass"
-      ><i
-        v-if="icon"
-        :class="'tm-btn__icon material-icons'"
-        aria-hidden="true"
-        >{{ icon }}</i
-      ><span v-if="value" class="tm-btn__value">{{ value }}</span></span
-    ></router-link
-  ><a v-else-if="type === 'anchor'" class="tm-btn"
-    ><span :class="btnClass"
-      ><i
-        v-if="icon"
-        :class="'tm-btn__icon material-icons'"
-        aria-hidden="true"
-        >{{ icon }}</i
-      ><span v-if="value" class="tm-btn__value">{{ value }}</span></span
-    ></a
-  ><button v-else :type="type" class="tm-btn">
-    <span :class="btnClass"
-      ><i
-        v-if="icon"
-        :class="'tm-btn__icon material-icons'"
-        aria-hidden="true"
-        >{{ icon }}</i
-      ><img
+  <router-link v-if="type === 'link'" :to="to" class="tm-btn" exact="exact">
+    <span :class="btnClass">
+      <i v-if="icon" :class="'tm-btn__icon material-icons'" aria-hidden="true">
+        {{ icon }}
+      </i>
+      <span v-if="value" class="tm-btn__value">{{ value }}</span>
+    </span>
+  </router-link>
+  <a v-else-if="type === 'anchor'" class="tm-btn">
+    <span :class="btnClass">
+      <i v-if="icon" :class="'tm-btn__icon material-icons'" aria-hidden="true">
+        {{ icon }}
+      </i>
+      <span v-if="value" class="tm-btn__value">{{ value }}</span>
+    </span>
+  </a>
+  <button v-else :type="type" class="tm-btn">
+    <span :class="btnClass">
+      <i v-if="icon" :class="'tm-btn__icon material-icons'" aria-hidden="true">
+        {{ icon }}
+      </i>
+      <img
         v-else-if="img"
         :src="img"
         :class="'tm-btn__img'"
         aria-hidden="true"
-      /><span v-if="value" class="tm-btn__value">{{ value }}</span></span
-    >
+      /><span v-if="value" class="tm-btn__value">{{ value }}</span>
```

Comment:
It improves but is still closing/opening sibilings

Reply:
What would you expect here?